.buildkite/scripts/make_image.sh

```diff
@@ -33,12 +33,25 @@ case "$BUILD_TYPE" in
         DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_testnet_${VERSION}"
         LATEST_TAG="latest-val-${IMAGE_ARCH}"
         ;;
+    "val_tpm")
+        echo "Doing a testnet validator with TPM image build for ${IMAGE_ARCH}"
+        DOCKER_BUILD_ARGS="--build-arg BUILDER_IMAGE=$BASE_IMAGE --build-arg RUNNER_IMAGE=$BASE_IMAGE --build-arg REBAR_BUILD_TARGET=docker_testval_tpm $DOCKER_BUILD_ARGS"
+        BASE_DOCKER_NAME="validator_tpm"
+        DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_testnet_${VERSION}"
```

Comment:
Also, sorry if this is a stupid question, but do these libraries work/exist on ARM64 as well as AMD64?

Reply:
Is it armv7 in Alpine packages? Then yes. However I didn't try to build it on ARM.